filemage/CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+# CHANGELOG - filemage
+
```

Comment:
```suggestion

## 1.0.0

* [Added] Initial version of Filemage integration.
```

Reply:
resolved in 743314bd136fa41a564d99d590471733f4969a6b